tests/test_codebase/test_mmaction/test_video_recognition.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from tempfile import NamedTemporaryFile, TemporaryDirectory
+
+import pytest
+import torch
+from mmengine import Config
+
+import mmdeploy.backend.onnxruntime as ort_apis
+from mmdeploy.apis import build_task_processor
+from mmdeploy.codebase import import_codebase
+from mmdeploy.utils import Codebase, load_config
+from mmdeploy.utils.test import SwitchBackendWrapper
+
+import_codebase(Codebase.MMACTION)
+
+model_cfg_path = 'tests/test_codebase/test_mmaction/data/model.py'
+model_cfg = load_config(model_cfg_path)[0]
+deploy_cfg = Config(
+    dict(
+        backend_config=dict(type='onnxruntime'),
+        codebase_config=dict(type='mmaction', task='VideoRecognition'),
+        onnx_config=dict(
+            type='onnx',
+            export_params=True,
+            keep_initializers_as_inputs=False,
+            opset_version=11,
+            input_shape=None,
+            input_names=['input'],
+            output_names=['output'])))
+
+onnx_file = NamedTemporaryFile(suffix='.onnx').name
+task_processor = build_task_processor(model_cfg, deploy_cfg, 'cpu')
+img_shape = (224, 224)
+num_classes = 400
+video = 'tests/test_codebase/test_mmaction/data/video/demo.mp4'
+
+
+@pytest.fixture
+def backend_model():
+    from mmdeploy.backend.onnxruntime import ORTWrapper
+    ort_apis.__dict__.update({'ORTWrapper': ORTWrapper})
+    wrapper = SwitchBackendWrapper(ORTWrapper)
+    wrapper.set(outputs={
+        'output': torch.rand(1, num_classes),
+    })
+
+    yield task_processor.build_backend_model([''])
+
+    wrapper.recover()
+
+
+def test_build_backend_model(backend_model):
+    assert isinstance(backend_model, torch.nn.Module)
+
+
+def test_create_input():
```

Comment:
test_create_input, test_build_dataset_and_dataloader and test_build_test_runner failed with:

```dotnetcli
E               FileNotFoundError: class `VideoDataset` in mmaction/datasets/video_dataset.py: file "tests/test_codebase/test_mmaction/data/ann.txt" does not 
exist                                                                                                                                                         
                                                                                                                                                              
../../anaconda3/envs/repo2.0/lib/python3.7/site-packages/mmengine/registry/build_functions.py:136: FileNotFoundError
```